src/nvim/eval/funcs.c

```diff
@@ -11299,11 +11300,11 @@ static void f_winrestcmd(typval_T *argvars, typval_T *rettv, FunPtr fptr)
 
   ga_init(&ga, (int)sizeof(char), 70);
   FOR_ALL_WINDOWS_IN_TAB(wp, curtab) {
-    sprintf((char *)buf, "%dresize %d|", winnr, wp->w_height);
+    snprintf((char *)buf, "%dresize %d|", winnr, wp->w_height);
```

Comment:
Where is the size?

Reply:
😂 whoops... I was fixing the lint and forgot to check if the build compiled